src/server/common/helpers/token/token-utils.test.js

```diff
@@ -0,0 +1,42 @@
+import { createToken, secrets } from './token-utils.js'
+import { token } from '@hapi/jwt'
+
+jest.mock('@hapi/jwt', () => ({
+  token: {
+    generate: jest.fn()
+  }
+}))
+
+describe('token-utils', () => {
+  const apiKey =
+    'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
+  const secret = apiKey.substring(apiKey.length - 36, apiKey.length)
+  const iss = apiKey.substring(apiKey.length - 73, apiKey.length - 37)
```

Comment:
For readability, I think we could steal from the gov notify docs here: 

![image](https://github.com/user-attachments/assets/e042f008-062f-4d14-ae6c-efc76c3b9826)

So:

```javascript
  const apiKey =
    'my_test_key-26785a09-ab16-4eb0-8407-a37497a57506-3d844edf-8d35-48ac-975b-e847b4f122b0'
  const secret = '3d844edf-8d35-48ac-975b-e847b4f122b0'
  const iss = '26785a09-ab16-4eb0-8407-a37497a57506'
  ```
  
  This also stops us from using the core logic of the function under test within the test - e.g. if we got a bug with where we set our boundaries & we repeat it in the test & implementation, then the test-as-is won't catch this. 
  
 If we're being specific about what we expect to extract, then the test will.